Changelog.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- Add support for `formData` parameters, which get converted into request data structures.
```

Comment:
I believe it's in master, and this will need to be rebased/changed to use those changes.